cocos/ui/scroll-view.ts

```diff
@@ -1487,7 +1525,8 @@ export class ScrollView extends ViewGroup {
 
     protected _handleMoveLogic (touch: Touch): void {
         this._getLocalAxisAlignDelta(this._deltaPos, touch);
-        this._processDeltaMove(this._deltaPos);
+        this._deltaAmount.add(this._deltaPos);
+        // this._processDeltaMove(this._deltaPos);
```

Comment:
It can be removed.

Reply:
> It can be removed.

我的个人习惯:  通过注释 保留改变之前的老代码. 这样便于后面跟踪bug 和了解之前是什么模样. 
从实际经验来看, 这种通过注释保留之前代码的方式 比查看git历史要快捷. 等功能测试没有问题 且上线一段时间之后, 再清理这种多余注释.
其实cocos目前的代码. 也有很多类似的注释.

Reply:
可以通过注释的方式描述下，不建议直接保留这种方式，这样对后面维护的人也很困惑。以后要清理基本上没有表示标识，很容易导致这种东西成为历史遗留。

Reply:
> 可以通过注释的方式描述下，不建议直接保留这种方式，这样对后面维护的人也很困惑。以后要清理基本上没有表示标识，很容易导致这种东西成为历史遗留。

已改 .删除了多余注释